website_supplier/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+================
+Website Suppliers
+================
```

Comment:
@pedrobaeza Hi pedro, this line doesn't come from my last commit. On my last commit, i had to replace it all into "Website Supplier List"